pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -419,6 +427,17 @@ func validateName(name string) error {
 	if err := validation.IsQualifiedName(name); err != nil {
 		return fmt.Errorf("name is not a valid Kubernetes label name: %v", err)
 	}
+	// we expect the resource property names to be in this format `[PREFIX]/[CAPACITY_TYPE]-[RESOURCE_NAME]`.
+	if strings.HasPrefix(name, resourcePropertyNamePrefix) {
+		resourcePropertyName, _ := strings.CutPrefix(name, resourcePropertyNamePrefix)
+		segments := strings.Split(resourcePropertyName, "-")
+		if len(segments) != 2 || len(segments[0]) == 0 || len(segments[1]) == 0 {
```

Comment:
Hi Arvind! We also have some requirements on the second argument (the resource name).